src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -278,6 +291,31 @@ export default {
       return publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
     },
 
+    viewURI(asset_id, name) {
+      const file_ext = this.fileExtension(name);
+      if (file_ext === 'nii') {
+        return `https://bioimagesuiteweb.github.io/alphaapp/viewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
+      if (file_ext === 'nwb') {
+        return `http://nwbexplorer.opensourcebrain.org/nwbfile=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
```

Comment:
I would prefer if code was separated from registry of services, so that registry could be reused later by multiple portals.
See https://github.com/datalad/datalad-deprecated/blob/master/datalad_deprecated/resources/website/assets/js/main.js#L21 and how used

Reply:
Note: we need to add maxsize field there and set it to eg 1GB or might be even less for bisweb. Sure to limitations extracted data must not exceed 4GB and anyways, download might be too slow etc. So I wouldn't bother exposing that service for known to fail out be problematic files